test/mocha.test.js

```diff
@@ -0,0 +1,70 @@
+var assert = require('chai').assert
+var request = require('supertest')
+require('../example/app.js')
+
+describe('Mongo-Throttle', function () {
+  // app.use('/api/', throttle())
+  describe('No Configs', function () {
+    it('Test', function (done) {
+      request('localhost:3000')
+        .get('/')
+        .expect(200, function (err, res) {
+          if (err) return done(err)
+          assert.equal(err, null)
+          console.log(res.headers['x-rate-limit-limit'])
```

Comment:
Yea im fine with that , i actually left these in cause my comp was dying and i had no more time to test so i just left it  👍